fe/fe-core/src/main/java/org/apache/doris/nereids/stats/ExpressionEstimation.java

```diff
@@ -172,6 +172,10 @@ public ColumnStatistic visitSlotReference(SlotReference slotReference, Statistic
     public ColumnStatistic visitBinaryArithmetic(BinaryArithmetic binaryArithmetic, Statistics context) {
         ColumnStatistic leftColStats = binaryArithmetic.left().accept(this, context);
         ColumnStatistic rightColStats = binaryArithmetic.right().accept(this, context);
+        if (leftColStats == null || rightColStats == null) {
+            return ColumnStatistic.UNKNOWN;
+        }
```

Comment:
why add this? this is not a right way. we should check why `leftColStats == null || leftColStats == null`, and let them return right value @englefly @Kikyou1997 please help to check it

Reply:
I ran into this problem during development, but then it didn't reappear. I currently removed this code